### DISCUSSION TITLE
请问有数据库sql，本地可以跑吗

### DISCUSSION THREAD

**Discussion by yhcxy** [halo-dev/halo#6400] [Q&A]
### System information

请问有数据库sql，本地可以跑吗

### What is the project operation method?

Docker

### What happened?

请问有数据库sql，本地可以跑吗

### Reproduce Steps

请问有数据库sql，本地可以跑吗

### Relevant log output

```shell
请问有数据库sql，本地可以跑吗
```

### Additional information

请问有数据库sql，本地可以跑吗

**Comment by ruibaby** [comment]
请至少确保先看过文档，https://docs.halo.run/developer-guide/core/run
